### PR TITLE
HOMETASK2 - 4. JS Basics 31.08.2023 19:15

### PR DESCRIPTION
**Pull request by rostykbest**

### CODE REVIEW

File: git_basics/readme.md
Comment (sttaran) [inline git_basics/readme.md:1]: why have you deleted content of this file?

File: js_basics/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+//const { name } = require("./test");
```
Comment (sttaran) [inline js_basics/index.js:1]: unused imports is bad practice even though it's commented

File: js_basics/index.js
Comment (sttaran) [inline js_basics/index.js:1]: this file is not a part of home work. So it should be excluded

File: js_basics/task1.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const v_string = "string";
```
Comment (sttaran) [inline js_basics/task1.js:1]: we should name variables using camelCase but not snake_case

File: js_basics/task5-1.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+let radius = 5.25;
+const pi = 3.14;
+let circleArea = pi * Math.pow(radius, 2);
+console.log(circleArea);
+console.log(Number.parseFloat(circleArea).toFixed(2));
```
Comment (sttaran) [inline js_basics/task5-1.js:5]: `Number.parseFloat` call is unnecessary since circleArea is type of number

File: js_basics/task5-1.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+let radius = 5.25;
+const pi = 3.14;
```
Comment (sttaran) [inline js_basics/task5-1.js:2]: use `Math.PI` instead

File: js_basics/test.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+//const name = "John";
```
Comment (sttaran) [inline js_basics/test.js:1]: unused file, it should be excluded

**Comment by sttaran** [comment]
PR should be merged only after review

File: git_basics/readme.md
Comment (rostykbest) [reply git_basics/readme.md:1]: actually I don't remember, anyway it's no use of it

File: js_basics/index.js
Comment (rostykbest) [reply js_basics/index.js:1]: deleted it

File: js_basics/index.js (hunk 1 above)
Comment (rostykbest) [reply js_basics/index.js:1]: ok, thanks

File: js_basics/task1.js (hunk 2 above)
Comment (rostykbest) [reply js_basics/task1.js:1]: fixed that

File: js_basics/task5-1.js (hunk 4 above)
Comment (rostykbest) [reply js_basics/task5-1.js:2]: I changed them all

File: js_basics/task5-1.js (hunk 3 above)
Comment (rostykbest) [reply js_basics/task5-1.js:5]: fixed it

File: js_basics/test.js (hunk 5 above)
Comment (rostykbest) [reply js_basics/test.js:1]: deleted it

**Pull request merged**